lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1042,7 +1042,7 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => true,
           :deprecated => true,
           :description => deprecated_description(
-            :'distributed_tracing-enabled',
```

Comment:
We don't need to leave the `-` because there's already something in `.deprecated_description` that converts the `.` to ` -` for the link.